holonote/app/tabulator.py

```diff
@@ -18,6 +18,7 @@ class AnnotatorTable(pn.viewable.Viewer):
     _updating = False
 
     def __init__(self, annotator, **params):
+        self.tabulator_kwargs = params.pop("tabulator_kwargs", {})
```

Comment:
Wondering whether it should simply be a `param`?

Reply:
Thanks for the feedback! I'm not sure of all the implications of doing it one way or another, but I suppose that making it a param makes it documentable. I just pushed the change